libraries/src/Console/RemoveUserFromGroupCommand.php

```diff
@@ -141,7 +141,9 @@ protected function doExecute(InputInterface $input, OutputInterface $output): in
 
 			if (\count(Access::getGroupsByUser($user->id, false)) < 2)
 			{
-				$this->ioStyle->error("Can't remove '" . $user->username . "' from group '" . $result . "'! Every user needs at least one group");
+				$this->ioStyle->error("Can't remove '" . $user->username . "' from group '" . $result . "
+					'! Every user needs to be a member of at least one group"
```

Comment:
```suggestion
				$this->ioStyle->error("Can't remove '" . $user->username . "' from group '" . $result
					. "'! Every user needs to be a member of at least one group"
```
I would not do the line break within the sting.

Reply:
I didnt want to either but it gave a line lenth error and this change is consistent with line 134

Reply:
@brianteeman In line 134 the new line is not done within the double quotes and so not within the string. It is exactly there like I suggest here, the line ends with a sting end (double-quote in this case) and the next line starts with the concatenation operator. See https://github.com/joomla/joomla-cms/blob/fc6d246d47c557ee6fb1702ec2d3ba3d97cd2484/libraries/src/Console/RemoveUserFromGroupCommand.php#L134 :

```
$this->ioStyle->error("Can't remove user '" . $user->username . "' from group '" . $result . "'! "
	. $result . " needs at least one active user!"
);
```

Reply:
Your code currently does the line change between the double quotes and so in the middle of a string and that's not the case in line 134.